src/main/backend/process.ts

```diff
@@ -147,11 +151,10 @@ export class OwnedBackendProcess implements BaseBackendProcess {
             return;
         }
 
+        await fetch(`${this.url}/shutdown`, { method: 'POST' });
         if (this.process.kill()) {
             this.process = undefined;
             log.info('Successfully killed backend.');
-        } else {
-            throw new Error('Unable to the backend process. Kill returned false.');
         }
```

Comment:
Why did you remove this?

Reply:
I think I did it assuming that it wouldn't always be able to be killed if it's sitting itself down now